src/watertap_contrib/seto/unit_models/surrogate/vagmd_surrogate.py

```diff
@@ -0,0 +1,1233 @@
+                    b.thermal_resistance_hot
+                    * (
+                        b.feed_props[0].temperature
+                        - b.condenser_in_props[0].temperature
+                    ),
+                    to_units=pyunits.kW,
+                )
+
+            @self.Constraint(
+                doc="Calculate inlet cooling water temperature in closed cooling mode"
+            )
+            def eq_cooling_in_temp(b):
+                return (
+                    b.cooling_in_props[0].temperature
+                    == b.feed_props[0].temperature
+                    - pyunits.convert(b.cooling_power_thermal, to_units=pyunits.W)
+                    / b.effectiveness_heat_exchanger
+                    / b.thermal_resistance_hot
+                )
+
+            @self.Constraint(
+                doc="Calculate outlet cooling water temperature in closed cooling mode"
+            )
+            def eq_cooling_out_temp(b):
+                return b.cooling_out_props[0].temperature == b.cooling_in_props[
+                    0
+                ].temperature + b.thermal_resistance_hot / b.thermal_resistance_cold * (
+                    b.feed_props[0].temperature - b.condenser_in_props[0].temperature
+                )
+
+        else:  # self.config.cooling_system_type == "open"
+
+            @self.Constraint(doc="Calculate cooling power requirment")
+            def eq_cooling_power_thermal(b):
+                return b.cooling_power_thermal == pyunits.convert(
+                    b.effectiveness_heat_exchanger
+                    * b.thermal_resistance_hot
+                    * (b.feed_props[0].temperature - b.cooling_in_props[0].temperature),
+                    to_units=pyunits.kW,
+                )
+
+            @self.Constraint(
+                doc="Calculate condenser inlet water temperature in open cooling mode"
+            )
+            def eq_condenser_in_temp(b):
+                return (
+                    b.condenser_in_props[0].temperature
+                    == b.feed_props[0].temperature
+                    - pyunits.convert(b.cooling_power_thermal, to_units=pyunits.W)
+                    / b.thermal_resistance_hot
+                )
+
+            @self.Constraint(
+                doc="Calculate outlet cooling water temperature in open cooling mode"
+            )
+            def eq_cooling_out_temp(b):
+                return (
+                    b.cooling_out_props[0].temperature
+                    == b.cooling_in_props[0].temperature
+                    + pyunits.convert(b.cooling_power_thermal, to_units=pyunits.W)
+                    / b.thermal_resistance_cold
+                )
+
+        # Set alias for state properties
+        FFR = pyunits.convert(
+            self.feed_props[0].flow_vol_phase["Liq"], to_units=pyunits.L / pyunits.h
+        )
+        S = self.feed_props[0].conc_mass_phase_comp["Liq", "TDS"]
+        TEI = self.evaporator_in_props[0].temperature
```

Comment:
I am not wild about acronyms as aliases from a readability perspective, but it is ok, just include in a comment what the acronyms stand for.

Reply:
Comments added.